spec/models/move_spec.rb

```diff
@@ -412,6 +412,50 @@
     end
   end
 
+  describe '#rebook' do
```

Comment:
could you add a test for when the move is already rebooked?

Reply:
Done 👍